lib/wallaroo/core/boundary/boundary.pony

```diff
@@ -410,6 +410,13 @@ actor OutgoingBoundary is Consumer
     to be sent but any writes that arrive after this will be
     silently discarded and not acknowleged.
     """
+    @printf[I32]("OutgoingBoundary dispose\n".cstring())
+
+    if _queue.size() > 0 then
```

Comment:
Should this be `if _queue.size() == 0`?

Reply:
No, this is...

"if we have queued items, don't dispose yet, requeue another dispose attempt".

Reply:
Oh I see.  I didn't realize this was in the `dispose()` behavior.  Nevermind.